src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -667,6 +667,30 @@ public async Task CloseAsync_ByServer_AcceptThrows()
             }
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task CloseAsync_MultipleCalls_FollowingCallsAreIgnored(bool client)
+        {
+
+            (QuicConnection clientConnection, QuicConnection serverConnection) = await CreateConnectedQuicConnection();
+
+            using (clientConnection)
+            using (serverConnection)
+            {
+                if (client)
+                {
+                    await clientConnection.CloseAsync(0);
```

Comment:
In the original commit, you also closed the other side. Was the original commit wrong or is it missing here or it doesn't matter? Otherwise, LGTM though, thanks.

Reply:
I figured it is not necessary, as the connection would be closed by the other side (and if not, it would be closed silently by `Dispose`)